config.toml

```diff
@@ -119,7 +119,7 @@ lastmod = ["lastmod", ":git", "date"]
 #  weight = 30
 
 [[menu.footer]]
-  url = "/contact"
+  url = "mailto:crownlabs.polito@gmail.com?Subject=CrownLabs"
```

Comment:
Just curiosity. Do we stick to this email or switch to the PoliTO one?

Reply:
Updated. Thanks, Marco.